Update javascript.remix.mdx (#7728)
getsentry/sentry-docs: src/platform-includes/getting-started-install/javascript.remix.mdx

We recommend installing the SDK through our installation wizard:

```bash
npx @sentry/wizard@latest -i remix
```

The wizard will prompt you to log in to Sentry. It will then automatically do the following steps for you:

- create two files in the root directory of your project, `entry.client.tsx` and `entry.server.tsx` (if they don't exist yet)

> - add the default Sentry.init() for the client and serverside SDK, in these files respectively

Reviewer: This kind of makes it sound like there are separate SDKs for the client and server. Maybe this is better?

```suggestion
- add the default `Sentry.init()` for the client in `entry.client.tsx` and the server in `entry.server.tsx`
```

- create `.sentryclirc` with an auth token to upload source maps (this file is automatically added to `.gitignore`)

If you use [Remix future flags](https://remix.run/docs/en/main/pages/api-development-strategy#current-future-flags), the wizard will instrument your application accordingly to support Remix v2 features.

After the wizard setup is completed, the SDK will automatically capture unhandled errors, and monitor performance.
You can also <PlatformLink to="/usage/">manually capture errors</PlatformLink>.

To learn how to connect your app to Sentry and deploy it on Vercel, see the [Vercel integration](/product/integrations/deployment/vercel/).

<Note>

> If the setup through the wizard doesn't work for you, you can also <PlatformLink to="/manual-setup/">set up the SDK manually</PlatformLink>.

Reviewer: This link doesn't work. In the Next docs, this links to a separate manual setup page, but that page doesn't exist for Remix.

Along those lines, the sections that follow on the Remix Getting Started page have those manual setup instructions, which will likely be confusing for users since some of that configuration will already have been done by the wizard.

Could we split this page off into a manual configuration page too like Next does?

Author: yes for sure, i was just trying to get something done faster :D 

could you help me with splitting it off?

We recommend installing the SDK through our installation wizard:

```bash
npx @sentry/wizard@latest -i remix
```

The wizard will prompt you to log in to Sentry. It will then automatically do the following steps for you:

- create two files in the root directory of your project, `entry.client.tsx` and `entry.server.tsx` (if they don't exist yet)
- in these files, add your initialization code for the client-side SDK and server-side SDK, respectively.
- create `.sentryclirc` with an auth token to upload source maps (this file is automatically added to `.gitignore`)

> - add an example page to your app to verify your Sentry setup

Reviewer 2: We don't add an example page with the current implementation.

Author: removed

> - add an example page to your app to verify your Sentry setup

Reviewer 2: ```suggestion
- instrument your `App` component and `ErrorBoundary` in your `root` route.
- modify your `package.json` scripts to generate and upload your sourcemaps to Sentry.
```

> The two configuration types are mostly the same, except that some configuration features, like Session Replay, only work in `entry.client.tsx`.

Reviewer 2: The main instrumentation work is done outside of `entry` files. So, might be better to mention v2 flags here instead.

```suggestion
If you use [Remix future flags](https://remix.run/docs/en/main/pages/api-development-strategy#current-future-flags), the wizard will instrument your application accordingly to support Remix v2 features. 
```
